packages/kotti-ui/source/kotti-field-select/components/GenericSelectField.vue

```diff
@@ -0,0 +1,434 @@
+<template>
+	<div class="kt-field-select">
+		<div ref="tippyTriggerRef">
+			<KtField
+				v-bind="{ field }"
+				:getEmptyValue="() => (isMultiple ? [] : null)"
+				:helpTextSlot="helpTextSlot"
+			>
+				<div
+					class="kt-field-select__input-and-tags"
+					@click.stop="handleInputClick"
+				>
+					<div
+						v-if="isMultiple && !field.isEmpty"
+						class="kt-field-select__tags"
+					>
+						<div
+							v-for="option in visibleSelectedTags"
```

Comment:
```suggestion
							v-for="tag in visibleSelectedTags"
```

Reply:
decided to keep the `option` because it is effectively a list of options, and using `tag.attributeThatBelongsToOption` might prove to be more confusing